doc/api.rst

```diff
@@ -26,21 +26,29 @@ ICLabel
 This is the model originally available for `EEGLab <https://github.com/sccn/ICLabel>`_.
 The model was ported from matconvnet using `pytorch <https://pytorch.org/>`_.
 
+ICLabel is designed to classify ICs fitted with an extended infomax ICA
+decomposition algorithm on EEG datasets referenced to a common average and
+filtered between [1., 100.] Hz.
```

Comment:
Perhaps we can say also something like:

We allow users to still run their EEG data if they don't meet these specifications, but we do not make any guarantees for software performance. Moreover, the ICLabel paper did not study the effects of these preprocessing steps.